README.md

```diff
@@ -34,7 +34,6 @@ module "security_hub_collector_runner" {
   s3_results_bucket = aws_s3_bucket.security_hub_collector.bucket
   s3_key            = ""
   team_map          = filebase64("${path.module}/teammap.json") // read more in Required Parameters
```

Comment:
add `scheduled_task_state` to the example?

Reply:
And update the "Optional Parameters" below? Or just delete all the parameters sections, I think it's just asking for drift to list those out and not update them in an automated way